crates/bevy_audio/src/audio.rs

```diff
@@ -141,30 +151,40 @@ impl PlaybackSettings {
         self.speed = speed;
         self
     }
+
+    /// Helper to enable or disable spatial audio.
+    pub const fn with_spatial(mut self, spatial: bool) -> Self {
+        self.spatial = spatial;
+        self
+    }
 }
 
-/// Settings for playing spatial audio.
+/// Settings for the listener for a spatial audio source.
 ///
-/// Note: Bevy does not currently support HRTF or any other high-quality 3D sound rendering
-/// features. Spatial audio is implemented via simple left-right stereo panning.
+
 #[derive(Component, Clone, Debug)]
-pub struct SpatialSettings {
-    pub(crate) left_ear: [f32; 3],
-    pub(crate) right_ear: [f32; 3],
-    pub(crate) emitter: [f32; 3],
+pub struct SpatialListener {
+    /// Left ear position relative to the `GlobalTransform`.
+    pub left_ear_offset: Vec3,
+    /// Right ear position relative to the `GlobalTransform`.
+    pub right_ear_offset: Vec3,
+}
+
+impl Default for SpatialListener {
+    fn default() -> Self {
+        Self::new(4.)
+    }
 }
 
-impl SpatialSettings {
+impl SpatialListener {
     /// Configure spatial audio coming from the `emitter` position and heard by a `listener`.
     ///
-    /// The `listener` transform provides the position and rotation where the sound is to be
-    /// heard from. `gap` is the distance between the left and right "ears" of the listener.
-    /// `emitter` is the position where the sound comes from.
-    pub fn new(listener: Transform, gap: f32, emitter: Vec3) -> Self {
-        SpatialSettings {
-            left_ear: (listener.translation + listener.left() * gap / 2.0).to_array(),
-            right_ear: (listener.translation + listener.right() * gap / 2.0).to_array(),
-            emitter: emitter.to_array(),
+    /// `gap` is the distance between the left and right "ears" of the listener. Ears are
+    /// positioned on the x axis.
```

Comment:
Great, there goes my awesome idea of an alien species game that have ears on their front and back sides! Pah! :)

Reply:
That's no problem -- you can still construct `SpatialListener` manually with your own ear offsets. This constructor is just for convenience with the most common use case.